src/utils/videofx/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
I guess videofx is not available through npm ?

Reply:
if not, maybe we can try using the Github URL directly with a tag, see https://stackoverflow.com/a/17509764

Reply:
I think not. It is a complete small project rather than an installable package. I repurposed the code and had to convert it manually from ts to js.
Some files need renaming, others need modifications. Therefore I think, it is neccessary to create a own version/fork of the code.